frontend/src/pages/meditation/components/modal-wrapper/styles.module.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  width: 100%;
+  height: 100vh;
+  background-image: url("../../../../assets/img/auth-background.png");
```

Comment:
First of all this component will be deleted after sidebar will be merged) In general if we use path from public folder, it will be broken after redirecting, so as I understand at now our approach to make it relative

Reply:
k, sounds good, thanks for the clarification